### ISSUE TITLE
Rework SCSS error display in Admin after Saving Styles

### ISSUE THREAD

**Issue by simmonsr** [gantry/gantry5#2533]
Can we rework how SCSS errors are displayed in the admin after saving Styles? Like a full page error instead? Currently I had this error display (which is no help to me) and so I had to selectively disable each of my loaded SCSS sheets to find that it was an issue with social particle SCSS.

![Screen Shot 2019-09-04 at 4 24 10 PM](https://user-images.githubusercontent.com/5002814/64295933-3e309400-cf38-11e9-9d2a-e9e8b8cdc157.jpg)

I did try disabling scripts in my inspector and viewing the errors that displayed, but several nodes just kept generating and so my inspector froze up.

**Comment by marktaylor46** [comment]
I totally agree with this. Once you have one error itusually generates a cascade of thousands of other errors. The error you actually need to fix is the very first one shown. It is possible to use a web inspector to find it but it's really hard work because the panel fades out after 5-10 seconds so you have to keep returning the mouse over the panel to reset that. I think perhaps adding a button to shown the whole list of errors on a seperate page would be a good idea. At the very least we need to be able to prevent the fade out of the error panel.
